client/src/common/styledComponents/List.js

```diff
@@ -0,0 +1,32 @@
+import styled from 'vue-styled-components';
+
+export const List = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+`;
+
+export const ListItem = styled.div`
+  flex: 1 0 calc(21%);
+  border: 1px solid #979797;
+  min-height: 200px;
+  margin: 0 10px 10px 0;
+  padding: 8px 14px;
+  font-size: 18px;
+  color: #222222;
+  display: flex;
+  flex-direction: column;
+
+  & div {
```

Comment:
Minor: Could we skip cascade usage here? If the answer is no, could we use a class instead?

Reply:
No problem :)